src/antares/service/local_services/study_local.py

```diff
@@ -10,13 +10,16 @@
 #
 # This file is part of the Antares project.
 
-from typing import Any, Optional
+from typing import TYPE_CHECKING, Any, Optional
 
 from antares.config.local_configuration import LocalConfiguration
 from antares.model.binding_constraint import BindingConstraint
 from antares.model.settings.study_settings import StudySettings
 from antares.service.base_services import BaseStudyService
 
+if TYPE_CHECKING:
```

Comment:
Why did you add this ? Was it to avoid circular imports ?

Reply:
Yes, that was to avoid circular imports with Study and the study services importing each other